app/helpers/sort_helper.rb

```diff
@@ -389,35 +392,43 @@ def find_filter_for_column(column, filter_mapping)
   # Some of the method arguments are only needed for specific actions.
   def action_menu(column, table_columns, caption, default_order, filter_column_mapping = {},
                   allowed_params: nil, **html_options)
-    caption ||= column.to_s.humanize
+    attribute = column.attribute
+    caption ||= attribute.to_s.humanize
 
-    filter = find_filter_for_column(column, filter_column_mapping)
+    filter = find_filter_for_column(attribute, filter_column_mapping)
     sortable = html_options.delete(:sortable)
 
     # `param` is not needed in the `content_arguments`, but should remain in the `html_options`.
     # It is important for keeping the current state in the GET parameters of each link used in
     # the action menu.
     content_args = html_options.merge(rel: :nofollow, param: nil)
 
-    render Primer::Alpha::ActionMenu.new(menu_id: "menu-#{column}") do |menu|
-      action_button(menu, caption, favorite: column == :favored)
+    render Primer::Alpha::ActionMenu.new(menu_id: "menu-#{attribute}") do |menu|
+      action_button(menu, column, caption, favorite: column == :favored)
 
       # Some columns are not sortable or do not offer a suitable filter. Omit those actions for them.
-      sort_actions(menu, column, default_order, content_args:, allowed_params:, **html_options) if sortable
-      filter_action(menu, column, filter, content_args:) if filter
+      sort_actions(menu, attribute, default_order, content_args:, allowed_params:, **html_options) if sortable
+      filter_action(menu, attribute, filter, content_args:) if filter
 
-      move_column_actions(menu, column, table_columns, content_args:, allowed_params:, **html_options)
-      add_and_remove_column_actions(menu, column, table_columns, content_args:, allowed_params:, **html_options)
+      move_column_actions(menu, attribute, table_columns, content_args:, allowed_params:, **html_options)
+      add_and_remove_column_actions(menu, attribute, table_columns, content_args:, allowed_params:, **html_options)
     end
   end
 
-  def action_button(menu, caption, favorite: false)
+  def action_button(menu, column, caption, favorite: false)
```

Comment:
This method seems to not have any tests, I would suggest adding some test just for the newly introduced functionality. Alternatively we could integraion test that all the icons are showing up correctly in the project headers.

Reply:
Good catch, I provided some specs 👍🏻